docs/design/ovs-pipeline.md

```diff
@@ -1,128 +1,217 @@
+| NXM_NX_REG8   | bits 0-11   | VLANIDField               |                |                                 | VLAN ID.                                                                                 |
+|               | bits 12-15  | CtZoneTypeField           | 0b0001         | IPCtZoneTypeRegMark             | Ct zone type is IPv4.                                                                    |
+|               |             |                           | 0b0011         | IPv6CtZoneTypeRegMark           | Ct zone type is IPv6.                                                                    |
+|               | bits 0-15   | CtZoneField               |                |                                 | Ct zone ID is a combination of VLANIDField and CtZoneTypeField.                          |
+| NXM_NX_XXREG3 | bits 0-127  | EndpointIP6Field          |                |                                 | Field to store IPv6 address of selected Service Endpoint.                                |
+
+Note that, reg marks that have overlapped bits will not be used at the same time, like `SwapField` and `PacketInTableField`.
+
+## Ct Marks
+
+We use some bits of OVS connection tracking mark to carry information throughout the pipeline. To enhance usability, we
+assign friendly names to the bits we use, which is called *ct mark*.
+
+| Field Range | Field Name            | CT Mark Value | CT Mark Name       | Description                                                     |
+|-------------|-----------------------|---------------|--------------------|-----------------------------------------------------------------|
+| bits 0-3    | ConnSourceCTMarkField | 0b0010        | FromGatewayCTMark  | Connection source is Antrea gateway port.                       |
+|             |                       | 0b0101        | FromBridgeCTMark   | Connection source is local bridge port.                         |
+| bit 4       |                       | 0b1           | ServiceCTMark      | Connection is for Service.                                      |
+|             |                       | 0b0           | NotServiceCTMark   | Connection is not for Service.                                  |
+| bit 5       |                       | 0b1           | ConnSNATCTMark     | SNAT'd connection for Service.                                  |
+| bit 6       |                       | 0b1           | HairpinCTMark      | Hair-pin connection.                                            |
+| bit 7       |                       | 0b1           | L7NPRedirectCTMark | Connection should be redirected to an application-aware engine. |
+
+## Ct Labels
+
+We use some bits of OVS connection tracking label to carry information throughout the pipeline. To enhance usability,
+we assign friendly names to the bits we use, which is called *ct mark*.
```

Comment:
same 2 comments as above (plus here there is a typo: *ct mark* instead of *ct label*)